action.yml

```diff
@@ -15,6 +15,10 @@ inputs:
   coverageToken:
     description: token to send with coverage data
     required: false
+  ignoredUsers:
+    description: 'disable action to run on specified users PRs'
+    default: ${['dependabot']}
```

Comment:
I haven't seen this syntax before. Is this how you specify a default value for an array?

Reply:
Tested with ['dependabot', 'adamTranquilla', 'neo-Lucas-Johannson'] and it skipped the action for me, I will double check tho

Reply:
Just going with a comma separated string. Seeing this as a common way to essentially have an array as a yaml value. updated the readme to reflect this

Reply:
this seems to be the default way to handle arrays